ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -490,6 +493,13 @@ message DamlContractKeyState {
   google.protobuf.Timestamp active_at = 2;
 }
 
+message DamlContractKeyIdPair {
+  DamlContractKey key = 1;
+
+  // Contract id the key was resolved to.
+  string resolved_to_id = 2;
```

Comment:
A contract key may also be resolved to "no contract". How do you represent such a resolution? If you use an empty string for this, it may be worth documenting here. Alternatively, you could use a one-of to distinguish the cases.

Reply:
I think an empty string is probably fine, in keeping with idiomatic protobuf behavior (where an empty string means "no value"). I do agree it needs documenting.

Reply:
Fixed.